fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -260,6 +260,21 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int label_keep_max_num = 1000;
 
+    /**
+     * StreamLoadTasks hold by StreamLoadMgr will be cleaned
+     */
+    @ConfField(mutable = true)
+    public static int stream_load_task_keep_max_num = 1000;
+
+    /**
+     * StreamLoadTasks of finished or cancelled can be removed
+     * 1. after *stream_load_task_keep_max_second*
+     * or
+     * 2. tasks total num > *stream_load_task_keep_max_num*
+     */
+    @ConfField(mutable = true)
+    public static int stream_load_task_keep_max_second = 3 * 24 * 3600; // 3 days
```

Comment:
3 days is too long?

Reply:
The value is the same as the default value of `label_keep_max_second`. Can we keep it for a longer period of time here? Because if the `keep_max_num is reached`, it can trigger cleanup. If the threshold is not reached, it is better to retain the historical records as much as possible for `show stream load` also get the result through the task